source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,379 @@
+   :class: singlecol
+
+Overview
+--------
+
+This guide details how to set up a Tiered Device Sync application while 
+this feature is in pre-alpha. This guide is intentionally not public,
+and is intended for internal MongoDB use.
+
+Create a Device Sync-Enabled App
+--------------------------------
+
+.. procedure::
+
+   .. step:: Create an App Services App
+
+      Create an :ref:`App Services App using the UI, CLI, or Admin API <create-app>`.
+      
+      For a quick proof-of-concept, :ref:`create an app from a 
+      template <create-template-app>`. If you select :guilabel:`Real-time Sync`
+      when you create a template app, this sets up a Device Sync-enabled 
+      task tracker app with a downloadable front-end client. This app comes
+      pre-configured with permissions for the user to write their own Items 
+      to an ``Item`` collection in a ``todo`` database. It also has 
+      :ref:`Development Mode <development-mode>` and 
+      :ref:`email/password authentication <email-password-authentication>` 
+      enabled.
+
+      If you want to define your own permissions and Device Sync settings,
+      create an app that does not use this template. Alternately, create a
+      template app but modify the settings.
+
+   .. step:: Enable Device Sync or Modify the Template Configuration
+
+      If you have not started with a template app, :ref:`enable Device Sync 
+      <enable-sync>` with Flexible Sync. Enable :ref:`Anonymous authentication 
+      <anonymous-authentication>`. Configure :ref:`permissions 
+      <device-sync-compatibility>` to allow anyone to read and write all data.
+
+      If you started with a template app, enable :ref:`Anonymous authentication 
+      <anonymous-authentication>`. Go to :guilabel:`Rules`, and delete the 
+      existing rule for the ``todo`` / ``Item`` collection. Replace it with 
+      ``readAndWriteAll``.
+
+Launch an EC2 Instance
+----------------------
+
+Launch an AWS EC2 instance to serve as the Tiered Device Sync server.
+This layer syncs device data between the devices and the EC2 instance, and then
+syncs data between the EC2 instance and your Atlas App Services app.
+
+.. procedure::
+
+   .. step:: Set Up an Ubuntu 18 Instance
+
+      Launch an EC2 instance running Ubuntu Server 18.x.
+
+   .. step:: Configure Required Network Settings
+
+      Under Network settings, add a rule to allow access via HTTP/Port 80.
+      For added security, restrict to your IP address.
+
+   .. step:: Provision and Launch the Instance
+
+      Select any other additional settings. Free tier settings are sufficient
+      for small PoC projects. Then, launch the instance.
+
+
+Enable Tiered Device Sync Authentication for Your App
+-----------------------------------------------------
+
+.. procedure::
+
+   .. step:: Get an Authorization Token
+
+      Get an authorization token for your App Services App. Log in to your 
+      App Services App, and run the following in your console:
```

Comment:
I'm just reading along but which console are you referring to?

Reply:
I guess you mean in the browser JavaScript console (`View > Developer > JavaScript Console` on chrome)

Reply:
Yup, that's right. I'll update to be clearer about this. 👍